.github/workflows/bump-packages.yaml

```diff
@@ -46,21 +60,4 @@ jobs:
           npm run bump-packages
           git add .
           git commit --no-allow-empty -m "chore(ci): bump packages" || true
-
-      - name: Create Pull Request
-        id: cpr
-        uses: peter-evans/create-pull-request@v6
-        with:
-          commit-message: 'chore(ci): bump packages'
-          branch: ci/bump-packages
-          title: 'chore(ci): bump packages'
-          body: |
-            - Bump package versions
-
-      - name: Merge PR
-        if: ${{steps.cpr.outputs.pull-request-operation == 'created'}}
-        env:
-          PULL_REQUEST_NUMBER: ${{steps.cpr.outputs.pull-request-number}}
-          GITHUB_TOKEN: ${{secrets.PAT}}
-        run: |
-          gh pr merge $PULL_REQUEST_NUMBER --squash --delete-branch
+          git push
```

Comment:
Pushing directly to main instead of creating a PR to align with our other workflows.